openmc/source.py

```diff
@@ -198,6 +198,43 @@ def to_xml_element(self):
             element.append(self.time.to_xml_element('time'))
         return element
 
+    def plot_energy(self, n_samples=10000, seed=None, axes=None):
```

Comment:
I think the convention for this case is to use `ax` or `axis` (not `axes` plural)
```suggestion
    def plot_energy(self, n_samples=10000, seed=None, ax=None):
```

Reply:
```suggestion
def plot_energy(self, n_samples=10000, seed=None, axes=None,  **kwargs):
```
How about ```axes``` as it is a matplotlib.axes.Axes object and tracks.py uses this arg name